docs/doc/10-deploy/06-metasrv/70-metasrv-cli-api.md

```diff
@@ -8,6 +8,14 @@ description:
 The binary `databend-meta` provides several handy commands to access the KVApi of databend meta service.
 `databend-meta --help` also includes a simple guide on using these CLI commands.
 
+**Caution**: these are low level API and should be avoided using it if possible:
```

Comment:
![image](https://user-images.githubusercontent.com/172204/202976481-d469b7c6-3239-427b-a911-2edd8b7da53b.png)
Pre-view: https://databend-git-fork-drmingdrmer-56-doc-databend.vercel.app/doc/deploy/metasrv/metasrv-cli-api

We can use the docusaurus `caution` to pretty it:
https://docusaurus.io/docs/next/markdown-features/admonitions